packages/accordions/examples/advanced.md

```diff
@@ -0,0 +1,194 @@
+### Accordion
+
+The Accordion component is implemented using the [W3C Accordion pattern](https://www.w3.org/TR/wai-aria-practices/#accordion).
+
+#### Controlled Usage
+
+This advanced example demonstrates custom behavior using a
+[controlled](https://reactjs.org/docs/forms.html#controlled-components) `Accordion`
+component. It also demonstrates additional header text and menu buttons in the `Accordion.Header`.
+
+```jsx
+const { SM } = require('@zendeskgarden/react-typography/src');
+const { Well } = require('@zendeskgarden/react-notifications/src');
+const { Toggle, Field, Label } = require('@zendeskgarden/react-forms/src');
+const { IconButton } = require('@zendeskgarden/react-buttons/src');
+const GearIcon = require('@zendeskgarden/svg-icons/src/16/gear-stroke.svg').default;
+const FolderIcon = require('@zendeskgarden/svg-icons/src/16/folder-open-stroke.svg').default;
+const { Dropdown, Menu, Item, Trigger } = require('@zendeskgarden/react-dropdowns/src');
+const { Button } = require('@zendeskgarden/react-buttons/src');
+
+const StyledSM = styled(SM)`
+  margin-left: ${props => props.theme.space.base}px;
```

Comment:
`font-weight: ${props => props.theme.fontWeights.regular};`

So that we don't have two bold type treatments next to each other in the example. This might be considered subjective, but the examples I've seen in product don't have descriptor/hint copy in bold.

Reply:
Sounds good, the font-weight has been updated.